### DISCUSSION TITLE
Adding more players

### DISCUSSION THREAD

**Discussion by parko210** [NeoArmageddon/co10_Escape#173] [Q&A]
Hi, looking to see if its possible to add more players slots?

**Comment by aussie-battler** [comment]
+ Load the mission.sqm file in Arma3 editor.
+ Find the player group & add more units to that group.
+ Save the mission,sqm
+ Copy & paste the mission.sqm into your escape folder.
+ Change description.ext file to reflect the number of units you added.

description.ext:

// Mission Header
class Header
{
  gameType = Escape;
  minPlayers = 1;
  maxPlayers = 10;  // change this line to the number of units you added
};
